.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,41 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Feature]: "
+labels: ["Feature"]
```

Comment:
As above, this should maybe be **kind/feature** (lower-case f)

Reply:
Updated to `kind/feature`